site-scrapers/LowellGeneral/index.js

```diff
@@ -29,20 +26,11 @@ function buildTimeSlotsUrl(stationIdAttr) {
     const locationId = stationIdAttr.split("_")[7];
 
     // start yesterday since they are looking for midnight of a given day
+    const midnight = "+00%3A00%3A00";
     const now = new Date(new Date().getTime() - 86400 * 1000);
```

Comment:
1) this isn't "now" - we should change the name
2) let's not mix Date() and moment() - we can use https://momentjscom.readthedocs.io/en/latest/moment/03-manipulating/02-subtract/ and do `moment().local().subtract(1, 'days')` to get yesterday.

Reply:
All set